optimize/util/optimize-commons/src/main/java/io/camunda/optimize/service/db/DatabaseConstants.java

```diff
@@ -10,24 +10,15 @@
 public final class DatabaseConstants {
 
   public static final String AGGREGATION_CONTAINS_NULL = "isNull";
```

Comment:
Unfortunately editing this file ran the formatter which created  all these whitespaces changes